commands/api.go

```diff
@@ -209,8 +212,8 @@ func apiCommand(cmd *Command, args *Args) {
 
 		params["query"] = query
 	} else {
-		path = strings.Replace(path, "{owner}", owner, 1)
-		path = strings.Replace(path, "{repo}", repo, 1)
+		path = strings.Replace(path, "{owner}", owner, -1)
+		path = strings.Replace(path, "{repo}", repo, -1)
```

Comment:
Could we use ReplaceAll instead?

Reply:
`ReplaceAll` only works with Go 1.12. I initially implemented this using ReplaceAll but tests running on older versions of Go broke in Travis.